tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
     pytest-cov
 
 commands =
-    jwt: pip install -e .[jwt]
+    jwt: pip install -e .[cryptography,jwt]
 
     flake8 .
-    pytest --cov
+    pytest {posargs:--cov}
```

Comment:
eh?

Reply:
then i run `tox -- -vx -k jwt_set` and pass through the flake args